sorc/orog_mask_tools.fd/orog_netcdf.fd/mask.fd/mask.f90

```diff
@@ -0,0 +1,164 @@
+ program mask_netcdf
+
+! Convert the UMD land use data to netcdf.
+
+ use netcdf
+
+ implicit none
+
+ integer*4, parameter :: idim=43200
+ integer*4, parameter :: jdim=21600
+ integer*4, parameter :: idim_p1=43201
+ integer*4, parameter :: jdim_p1=21601
+
+ character(len=150) :: filenetcdf, fileraw
+
+ integer :: i, istat, ncid, status, dim_i, dim_j
+ integer :: dim_ip1, dim_jp1
+ integer :: id_lon, id_lat, id_data
+ integer :: id_lat_corner, id_lon_corner
+
+ integer(kind=1), allocatable :: mask(:,:)
+
+ real(kind=8), allocatable :: lats(:), lons(:)
+ real(kind=8), allocatable :: lats_corner(:), lons_corner(:)
+ real(kind=8)    :: lat11, lon11, dx, dy
+
+ dx = 1.0_8/120.0_8
+ dy = -(1.0_8/120.0_8)
+
+ lat11 =  90.0_8 + dy*0.5_8
+ lon11 = -180.0_8 + dx*0.5_8
+
+ allocate(lons(idim),lats(jdim),mask(idim,jdim))
+ allocate(lons_corner(idim_p1),lats_corner(jdim_p1))
+
+ do i = 1, idim
+   lons(i) = real((i-1),8) * dx + lon11
+   print*,'lon ',i,lons(i)
+ enddo
+
+ do i = 1, jdim
+   lats(i) = real((i-1),8) * dy + lat11
+   print*,'lat ',i,lats(i)
+ enddo
+
+ lat11 =  90.0_8
+ lon11 = -180.0_8
+
+ do i = 1, idim_p1
+   lons_corner(i) = real((i-1),8) * dx + lon11
+   print*,'lon_corner ',i,lons_corner(i)
+ enddo
+
+ do i = 1, jdim_p1
+   lats_corner(i) = real((i-1),8) * dy + lat11
+   print*,'lat_corner ',i,lats_corner(i)
+ enddo
+
+ fileraw="/scratch1/NCEPDEV/global/glopara/fix/raw/orog/landcover30.fixed"
+
+ open(11, file=trim(fileraw), access='direct', recl=idim*jdim)
+ read(11, rec=1, iostat=istat) mask
+ if (istat /= 0) stop 99
+ close(11)
+
+ print*,'mask ', maxval(mask),minval(mask)
+ where(mask > 0) mask = 1
+
+ filenetcdf="./landcover30.fixed.nc"
```

Comment:
maybe we could take this opportunity to have a more descriptive file name? like landcover.umd.30s.nc or something like that